src/microbe_stage/AgentProjectile2.tscn

```diff
@@ -0,0 +1,64 @@
+[gd_scene load_steps=11 format=2]
```

Comment:
This doesn't seem to be used anywhere?
I don't think an unused scene should be included in the main branch unless this is moved somewhere like `test/experimental_scenes`